src/commands/schemas.ts

```diff
@@ -79,3 +108,51 @@ async function loadOrCreateSchemaViewer(schema: Schema) {
   vscode.languages.setTextDocumentLanguage(textDoc.document, schema.fileExtension());
   return textDoc;
 }
+
+/**
+ * Get the highest versioned schema(s) related to a single topic from the schema registry
+ * as decided by TopicNameStrategy. May return two schemas if the topic has both key and value schemas.
+ */
+export async function getLatestSchemasForTopic(topic: KafkaTopic): Promise<Schema[]> {
+  if (topic.isLocalTopic()) {
+    throw new Error("Cannot get schemas for local topics (yet)");
+  }
```

Comment:
Is it possible to hide this action for local topics instead of throwing an error when they try it?

Reply:
Maybe better explicitly using `ccloud-kafka-topic` like:
```json
"when": "view == confluent-topics && viewItem =~ /ccloud-kafka-topic.*-with-schema.*/",
```
in here until we support local schemas (soon)?:
https://github.com/confluentinc/vscode/blob/b7e86921761d8f86b275e02fad10a467813d0bd6/package.json#L592

Reply:
It is hidden for local topics. This exception here is just the interior function protecting its assumptions.

The regex in package json *already* doesn't bind to local topics --- because they don't have schemas.